packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -787,29 +811,122 @@ test.serial('Settlement for unknown transaction (operator down)', async t => {
   const opDown = makeCustomer('Otto', cctp, txPub.publisher, feeConfig);
 
   const bridgePos = snapshot();
-  const sent = await opDown.sendFast(t, 20_000_000n, 'osmo12345');
+  const EUD = 'osmo10tt0';
+  const mintAmt = 5_000_000n;
+  const sent = await opDown.sendFast(t, mintAmt, EUD);
   await mint(sent);
   const bridgeTraffic = since(bridgePos);
 
   t.like(
     bridgeTraffic.bank,
     [
       {
-        amount: '20000000',
+        amount: String(mintAmt),
         sender: 'faucet',
         type: 'VBANK_GRAB',
       },
       {
-        amount: '20000000',
+        amount: String(mintAmt),
         recipient: 'agoric1qyqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc09z0g',
         type: 'VBANK_GIVE',
       },
     ],
     '20 USDC arrive at the settlement account',
   );
-  t.deepEqual(bridgeTraffic.local, [], 'no IBC transfers');
 
   await transmitTransferAck();
+  t.deepEqual(bridgeTraffic.local, [], 'no IBC transfers');
+
+  // activate oracles and submit evidence; expect Settler to forward (slow path)
+  // 'C12 - Contract MUST only pay back the Pool (fees) only if they started the advance before USDC is minted',
+  operators[0].setActive(true);
+  operators[1].setActive(true);
+  // set the 3rd operator to inactive so it doesn't report a 2nd time
+  operators[2].setActive(false);
+
+  // compute nonce from initial report so a new txId is not generated by `sendFast` helper
+  const nonce = Number(sent.txHash.slice(2));
+  await opDown.sendFast(t, mintAmt, EUD, nonce);
+
+  const [outgoingForward] = since(bridgePos).local;
+  t.like(outgoingForward, {
+    type: 'VLOCALCHAIN_EXECUTE_TX',
+    address: addresses.settlementAccount,
+    messages: [
+      {
+        '@type': '/ibc.applications.transfer.v1.MsgTransfer',
+      },
+    ],
+  });
+  const [outgoingForwardMessage] = outgoingForward.messages;
+  t.is(
+    outgoingForwardMessage.token.amount,
+    String(sent.tx.amount),
+    'full amount is transferred via `.forward()`',
+  );
+
+  const forwardInfo = JSON.parse(outgoingForwardMessage.memo).forward;
+  t.is(forwardInfo.receiver, EUD, 'receiver is osmo10tt0');
+
+  // in lieu of transmitTransferAck so we can set a nonce that matches our initial Advance
```

Comment:
considered parameterizing like: `transmitTransferAck(-1)` but came up short. I think it's only the sequence that difference, not the index of `localBridgeMessage`.